workflows/mag.nf

```diff
@@ -624,6 +632,17 @@ workflow MAG {
         ch_versions = ch_versions.mix(PRODIGAL.out.versions.first())
     }
 
+    /*
+    ================================================================================
+                                    Virus identification
+    ================================================================================
+    */
+
+    if (params.run_genomad){
```

Comment:
One last little one - should we now call this `params.run_virus_identification`?

Reply:
Yep, that makes sense to me. Sorry for missing that on the last pass!